jetty-proxy/src/test/java/org/eclipse/jetty/proxy/ProxyServletTest.java

```diff
@@ -185,6 +197,26 @@ private HttpClient prepareClient() throws Exception
         return result;
     }
 
+    private static HttpServletRequest mockClientRequest(String protocol)
+    {
+        return new org.eclipse.jetty.server.Request(null, null)
+        {
+            @Override
+            public String getProtocol()
+            {
+                return protocol;
+            }
+        };
+    }
+
+    private static HttpRequest mockProxyRequest()
+    {
+        return new HttpRequest(new HttpClient(), null, URI.create("https://example.com"))
+        {
+
+        };
+    }
+
```

Comment:
Inline these 2 methods, they are only used once.

Reply:
Doing so obscures the point of the test, and I'd really rather not. Though currently only used once, they make the test more readable.

Reply:
Nope. The test is _less_ readable and using mocks obscures the test. Remove them.